src/amp-story-player/amp-story-player-impl.js

```diff
@@ -14,12 +14,15 @@
  * limitations under the License.
  */
 
+import * as ampCaches from '../../build-system/global-configs/caches.json';
```

Comment:
Importing from `build-system` isn't supported, because isn't not included as one of our files when we pass them to closure.

Additionally, you can't import JSON like this in real JS, this is a node behavior without real semantics. TC39 is currently discussing JSON imports, but semantics we have will only expose a default export:

```js
import ampCaches from '…/caches.json' with type: 'json';

ampCaches.caches.map(…)
```

I'd imagine closure will have undefined semantics for this in the meantime. We'd need to preprocess this file into a JS file during builds, and you can import from there.